src/Features/Core/Portable/ExtractInterface/AbstractExtractInterfaceService.cs

```diff
@@ -177,6 +221,47 @@ public ExtractInterfaceResult ExtractInterfaceFromAnalyzedType(ExtractInterfaceT
                 navigationDocumentId: interfaceDocumentId);
         }
 
+        private async Task<ExtractInterfaceResult> ExtractInterfaceToSameFileAsync(
+            Solution solution, INamedTypeSymbol extractedInterfaceSymbol, SyntaxNode typeNode, ProjectId projectId,
+            DocumentId documentId, INamedTypeSymbol typeToExtractFrom, IEnumerable<ISymbol> includedMembers, CancellationToken cancellationToken)
+        {
+            var symbolToDeclarationAnnotationMap = CreateSymbolToDeclarationAnnotationMap(
```

Comment:
now with tuple support, do you need this "ref", out and etc? you can just return (map, solution, documentIds, annotations)

Reply:
Will do